### ISSUE TITLE
Admin Product edit - Description box stay collapsed 

### ISSUE THREAD

**Issue by joybuggycom** [thirtybees/thirtybees#1698]
https://github.com/thirtybees/thirtybees/blob/c790aa5c1efdd50a859c6dd8dde9bfd4be91f0ca/js/admin.js#L655

 Adding this part of code : 

```
	setInterval(() => {
		tinySetup({
			editor_selector: 'autoload_rte',
			autoresize_min_height: 200,
		});
	}, 1000);
```

Will fix the issue where the admin product description boxes keeps being collapsed.

**Comment by joybuggycom** [comment]
You might also need to add this to the CSS (we added it to the override.css) 


.translatable-field iframe {
    min-height: 200px;
    overflow-y: auto;
}

**Comment by getdatakick** [comment]
Thanks for suggestion. This is not the right place to fix this, thought. File `admin.js` can be loaded for pages that do not include `tinymce.inc.js`, which would cause javascript errors.

**Comment by joybuggycom** [comment]
> Thanks for suggestion. This is not the right place to fix this, thought. File `admin.js` can be loaded for pages that do not include `tinymce.inc.js`, which would cause javascript errors.

I see, is this something your going to add to the core?

**Comment by getdatakick** [comment]
Fixed in commit 756fd1d48d8079230cbed690102f3cdb3c102ed5.

It is now also possible to override default configuration for tinyMCE. You can do that by declaring global variable `window.tinymce_override_config`
